### PR TITLE
Epic/563 changes to character section of application form

### PR DESCRIPTION
**Pull request by mbrookeswebdev**

### CODE REVIEW

**Review by lloback — APPROVED** [review]
I think this is good to go - but you should fix some bugs that I will post on the PR - please move them to a ticket.

**Comment by lloback** [comment]
image

**Comment by lloback** [comment]
image

**Comment by lloback** [comment]
@mbrookeswebdev don't merge this yet.

**Review by lloback — CHANGES_REQUESTED** [review]
Don't merge this yet, because we need to coordinate this with the BA, other people. etc

File: src/components/RepeatableFields/CharacterInformation/LateTaxReturnDetails.vue [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    },
+    hint1: {
+      required: false,
+      type: String,
+      default: 'e.g. 2021/2022',
+    },
+    hint2: {
+      required: false,
+      type: String,
+      default: 'Please add details of your late tax return and provide details of any penalties or ' +
```
[the remark this answers is not in the document]
Comment (mbrookeswebdev) [reply src/components/RepeatableFields/CharacterInformation/LateTaxReturnDetails.vue:25]: Done in ticket #668

File: src/components/RepeatableFields/CharacterInformation/HmrcFineDetails.vue [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    },
+    hint1: {
+      required: false,
+      type: String,
+      default: 'e.g. 2021/2022',
+    },
+    hint2: {
+      required: false,
+      type: String,
+      default: 'Please add details of the incident and provide details of any penalties or ' +
```
[the remark this answers is not in the document]
Comment (mbrookeswebdev) [reply src/components/RepeatableFields/CharacterInformation/HmrcFineDetails.vue:25]: done in ticket #668

**Comment by mbrookeswebdev** [comment]
I attached a hand in ticket #668

**Comment by lloback** [comment]
Is this ready to go?

**Review by lloback — APPROVED** [review]

**Comment by mbrookeswebdev** [comment]
Not quite, next week we will have a meeting and it is possible other changes will be requested.

**Comment by lloback** [comment]
It is growing in size, and it is difficult to continue reviewing - 'agile' is small changes constantly and consistently!

**Comment by mbrookeswebdev** [comment]
Agreed, I think it is a large epic!

**Review by warrensearle — COMMENTED** [review]
Added some comments for you

File: src/helpers/constants.js [hunk 3]
```diff
@@ -42,11 +42,42 @@ const DEFAULT = {
...
+const DECLARATION1 = `I understand that it is my continuing duty, throughout the time that my application is being considered,
+                      to inform the Selection Exercise Team of all relevant matters that might affect the acceptability of my
+                      application when considered against this guidance.`;
+
+const DECLARATION2 = `I understand that failure to declare on the application form relevant information which later comes
+                      to light is likely to rule out further consideration of my application.`;
+
+const DECLARATION3 = `I also understand that failure to disclose information additional to that included on my application for,
+                      upon the request of the JAC and to the timetable requested by the JAC, may also rule out further consideration
+                      of my application.`;
```
[the remark this answers is not in the document]
Comment (lloback) [reply src/helpers/constants.js:54]: it makes sense.

**Review by warrensearle — APPROVED** [review]
This is a good little piece of work :) 👍 
Let's get it tested.

**Pull request merged**